.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: Build Releases
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - 'master'
+      - 'develop'
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+      - name: "Create filename"
+        id: fn
+        shell: bash
+        run: echo "FILENAME=build/rctab_${{ github.ref_name }}_${{ runner.os }}.zip" >> $GITHUB_OUTPUT
+          
+      - uses: actions/checkout@v3
+
+      - name: "Set up JDK 17.0.2"
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17.0.2'
+          distribution: 'temurin'
+
+      - name: "Validate Gradle wrapper"
+        uses: gradle/wrapper-validation-action@ccb4328a959376b642e027874838f60f8e596de3
+
+      - name: "Create zip with Gradle"
+        uses: gradle/gradle-build-action@67421db6bd0bf253fb4bd25b31ebb98943c375e1
```

Comment:
Where'd you pull this hash from? Is there a list somewhere to choose from?

Reply:
From the Getting Started demos from github: https://github.com/actions/starter-workflows/blob/main/ci/gradle-publish.yml

The list would be any commit here: https://github.com/gradle/gradle-build-action/commits/main

Reply:
Interesting, ok. Is this another thing we'd need to update regularly (i.e. before each release?).

Reply:
I don't think so -- perhaps if gradle has major security updates?

We can also have it use the latest commit, but locking the version is more secure and more reliable.